src/toil/test/cwl/cwlNewTest.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (C) 2015 Curoverse, Inc
```

Comment:
Is this still accurate? Should it also be (C) 2020 UCSC regents now?

Reply:
Will change.